modules/ossm-about-cert-manager.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-cert-manager-integration-istio_{context}"]
+= About integrating Service Mesh with cert-manager and istio-csr
+//TP1 content influx. Title, etc may change.
+
+The cert-manager tool provides integration with Istio through an external agent called `istio-csr`. The `istio-csr` agent handles certificate signing requests (CSR) from Istio proxies and the `controlplane` by performing the following:
+
+. Verifying the identity of the workload.
+. Creating a CSR through `cert-manager` for the workload.
+
+The cert-manager tool then creates a CSR to the configured CA Issuer which signs the certificate.
+
+[NOTE]
+====
+Red Hat provides support for integrating with `istio-csr` and `cert-manager`. Red Hat does not provide direct support for the `istio-csr` or the community `cert-manager` components. The use of community cert-manager shown here is for demonstration purposes only.
+====
+
+//For Istio users, cert-manager also provides integration with `istio-csr`, which is a certificate authority (CA) server that handles certificate signing requests (CSR) from Istio proxies. The server then delegates signing to cert-manager, which forwards CSRs to the configured CA server.
+
+.Prerequisites
+* One of these versions of cert-manager:
+** Red Hat cert-manager Operator 1.10 or later
```

Comment:
```suggestion
** {cert-manager-operator} 1.10 or later
```

Reply:
The cert-manager-operator attribute is not part of the Service Mesh stand alone common-attributes file. There is a separate Jira to track all the attributes that need to be added to the file for OSSM 3 GA.